src/main/java/net/sf/jsqlparser/statement/ExplainStatement.java

```diff
@@ -11,9 +11,16 @@
 
 import net.sf.jsqlparser.statement.select.Select;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * An {@code EXPLAIN} statement
+ */
 public class ExplainStatement implements Statement {
 
     private Select select;
+    private List<Option> options;
```

Comment:
I don't know exactly if an option can be here multiple times. So if then a Map or Set would be better here. Due to direct access to this list one is able to build inconsistent multiple options of the same type using getOptions or setOptions.

Reply:
Umm yeah, I think you're right, it can only be a LinkedSet (order preserved). I'll change that up!